src/main/scala/org/globalforestwatch/summarystats/annualupdate/AnnualUpdateSummary.scala

```diff
@@ -25,13 +25,15 @@ object AnnualUpdateSummary {
   // TreeLossSummary form Raster[TreeLossTile] -- cell types may not be the same
 
   implicit val mdhCellRegisterForTreeLossRaster1
-    : CellVisitor[Raster[AnnualUpdateTile], AnnualUpdateSummary] =
-    new CellVisitor[Raster[AnnualUpdateTile], AnnualUpdateSummary] {
+    : GridVisitor[Raster[AnnualUpdateTile], AnnualUpdateSummary]
+      = new GridVisitor[Raster[AnnualUpdateTile], AnnualUpdateSummary] {
+      private var acc: AnnualUpdateSummary = new AnnualUpdateSummary()
 
-      def register(raster: Raster[AnnualUpdateTile],
+      def result: AnnualUpdateSummary = acc
+
+      def visit(raster: Raster[AnnualUpdateTile],
```

Comment:
Codacy found an issue: [Method is longer than 50 lines](https://app.codacy.com/manual/thomas-maschler/gfw_forest_loss_geotrellis/pullRequest?prid=6292950)